index.xml

```diff
@@ -8626,6 +8748,11 @@ context.setColorEnforcedRange(-1, 255, 256);</x:codeblock>
               <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>
               <span class='rfc2119'>MUST</span> be a subset of the interface's
               <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>.
+              Similarly, if <a class='xattr' href='#Exposed'>[Exposed]</a> appears on both a
+              namespace and one of its namespace members, then the namespace member's
+              <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>
+              <span class='rfc2119'>MUST</span> be a subset of the namespace's
+              <a class='dfnref' href='#dfn-exposure-set'>exposure set</a>.
```

Comment:
I'm not sure how to square that with "Any extended attribute specified on a partial interface definition is considered to appear on the interface itself." It seems like the above snippet somehow makes Foo only exposed in Workers, per that text... (This is similar to my above reply.)